gather.go

```diff
@@ -9,7 +9,7 @@ import (
 	"net"
 	"time"
 
-	"github.com/danderson/nat/stun"
+	"github.com/chripell/nat/stun"
```

Comment:
Can you change these back?

The way I usually hack on forked github projects, is to pull the original project, then `git add remote fork <url of my fork>`. That way, I can keep building with the same paths, and easily switch between upstream and my own copy as needed.